### PR TITLE
Adding validation/label collation, a few edits to the instructions

### PR DESCRIPTION
**Pull request by steeleb**
Heya!

First pass in preparation for making our bare-bones classification model!

Many of the LOC here are actually lines of data, so this may look huge, but it should be pretty manageable. We generally avoid putting data in our repos, but I think it makes sense here. If you think all the data should live in Drive, please say so and I'll edit the code for that. We shouldn't hit any file size issues keeping the data on GH. 

This is progress on #25, #26, #42, #43. Only two scripts were modified. A number of GROD scripts were deleted (noted for each file).

## docs/index.Rmd

Just made a few edits here for clarity, in response to suggestions, and to [hopefully] cure any future issues that I've encountered while collating data. 

## eePlumB/2_data_ingestion/src/1_data_download.Rmd

Adapted this script to work with how are data are stored by users in Drive. I also did some prelim filtering. Note, I dropped all of the students from the upstream list, as only a few have done the validation and none have done labeling. Granted, only HD and I have labeled up until this point. 

WHEE!

B

### CODE REVIEW

File: docs/validation.Rmd
Comment (steeleb) [inline docs/validation.Rmd:1]: grod script

File: eePlumB/1_user_interface_script/assessment_reclassify10percent.js
Comment (steeleb) [inline eePlumB/1_user_interface_script/assessment_reclassify10percent.js:1]: grod script

File: docs/validation.html
Comment (steeleb) [inline docs/validation.html:1]: grod script

File: eePlumB/1_user_interface_script/readme_assessment_reclassify10percent.md
Comment (steeleb) [inline eePlumB/1_user_interface_script/readme_assessment_reclassify10percent.md:1]: grod script

File: eePlumB/1_user_interface_script/readme_reclassify_uncertain_category_js.md
Comment (steeleb) [inline eePlumB/1_user_interface_script/readme_reclassify_uncertain_category_js.md:1]: grod script

File: eePlumB/1_user_interface_script/reclassify_uncertain_category.js
Comment (steeleb) [inline eePlumB/1_user_interface_script/reclassify_uncertain_category.js:1]: grod script

File: eePlumB/1_user_interface_script/replace_duplicates.js
Comment (steeleb) [inline eePlumB/1_user_interface_script/replace_duplicates.js:1]: grod script

File: eePlumB/2_data_ingestion/data/in/placeholder.txt
Comment (steeleb) [inline eePlumB/2_data_ingestion/data/in/placeholder.txt:1]: removed placeholder

File: eePlumB/2_data_ingestion/data/out/placeholder.txt
Comment (steeleb) [inline eePlumB/2_data_ingestion/data/out/placeholder.txt:1]: removed placeholder

File: eePlumB/2_data_ingestion/data/tmp/placeholder.txt
Comment (steeleb) [inline eePlumB/2_data_ingestion/data/tmp/placeholder.txt:1]: removed placeholder

**Review by lindsayplatt — APPROVED** [review]
My only question is whether or not comitting those data files is necessary. Your RMD file, `eePlumB/2_data_ingestion/src/1_data_download.Rmd` has the code that downloads the data, so it seems like we could omit their inclusion in the actual repo since the _instructions_ for gathering them are committed.

**Comment by steeleb** [comment]
Love it. I'm going to merge this, then edit some things on my end to remove the data files from here. It would be good for us to have copies of all the raw data from folks (you know, when folks go and delete their eePlumB folders), so I'll make a folder for all the downloads in Drive and the collation on the next round. 

Thanks much!
B

**Pull request merged**